src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -30,12 +33,28 @@ public OperatorInterface(final SubsystemManager subMan) {
                 .whenReleased(new StopIntakeCommand(subsystemManager.getIntakeSubsystem(), subsystemManager.getElevatorSubsystem()))
                 .add();
 
-        manager.addButton(RobotConstants.BUTTONS.SHOOT_BUTTON)
+        manager.addButton(RobotConstants.BUTTONS.START_SHOOTER_BUTTON)
                 .whenPressed(new StartShooterCommand(subsystemManager.getShooterSubsystem(), subsystemManager.getElevatorSubsystem()))
                 .whenReleased(new StopShooterCommand(subsystemManager.getShooterSubsystem()))
                 .add();
+        
+        manager.addButton(RobotConstants.BUTTONS.FIRE_BUTTON)
+                .whenPressed(subsystemManager.getElevatorSubsystem().start())
+                .whenReleased(subsystemManager.getElevatorSubsystem().stop())
+                .add();
+
 
+        
         drive = subsystemManager.getDriveSubsystem();
+        
+        manager.addButton(4)
+                .whenPressed(new DriveForwardSetDistance(drive,48))
```

Comment:
when a command only needs one subsystem, i think we should prefer to have a method on the subsystem that produces the command, similar to what's being done for the elevatorsubsystem. Butt hat's just stylistic